sw/device/lib/testing/aes_testutils.c

```diff
@@ -0,0 +1,17 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/aes_testutils.h"
+
+#include "sw/device/lib/dif/dif_aes.h"
+#include "sw/device/lib/testing/check.h"
+#include "sw/device/lib/testing/rand_testutils.h"
+
+void aes_testutils_wait_for_status(dif_aes_t *aes, dif_aes_status_t flag,
+                                   bool value) {
+  bool status;
+  do {
+    CHECK_DIF_OK(dif_aes_get_status(aes, flag, &status));
```

Comment:
Should you use the new spin wait instead?

Reply:
Yes, one step at a time :) I made this PR much before the spinwait one. I will wait for that one to merge.